src/Core/Scheduler/Providers/WFS_Provider.js

```diff
@@ -4,61 +4,132 @@
  * Description: Provides data from a WFS stream
  */
 
-
+import Extent from '../../Geographic/Extent';
 import Provider from './Provider';
 import Fetcher from './Fetcher';
 import CacheRessource from './CacheRessource';
+import GeoJSON2Three from '../../../Renderer/ThreeExtended/GeoJSON2Three';
 
-/**
- * Return url wmts MNT
- * @param {Object} options
- * @param {?string} options.url - service base url
- * @param {?string} options.layer - requested data layer
- * @param {?string} options.format - image format (default: format/jpeg)
- * @constructor
- */
-// TODO should be refit to adhere the same logic as other providers (ie one instance for all layers)
-function WFS_Provider(options) {
+function WFS_Provider() {
     this.cache = CacheRessource();
-    this.baseUrl = options.url || '';
-    this.networkOptions = options.networkOptions;
-    this.layer = options.layer || '';
-    this.typename = options.typename || '';
-    this.format = options.format === undefined ? 'json' : options.format;
-    this.epsgCode = options.epsgCode || 4326;
+    this.batiments = [];
```

Comment:
`this.batiments` is unused

Do you have a mechanism to avoid duplicates geometry, because WFS queries return data that overlap extents:

see dirty patch, with `this.batiments`

https://github.com/iTowns/itowns/pull/195/files#diff-96c0e22190f6d3f57d176e91dfd7d1ffR130